### PR TITLE
docs: paralus adopter

### PR DESCRIPTION
**Pull request by vinckr**
FYI @techmaharaj

Screenshot 2022-08-10 at 09 52 03

### CODE REVIEW

File: templates/repository/common/ADOPTERS.md [hunk 1]
```diff
@@ -176,6 +176,12 @@ that your company deserves a spot here, reach out to
             <td align="center"><img height="32px" src="https://raw.githubusercontent.com/ory/meta/master/static/adopters/connctd.svg" alt="Connctd"></td>
             <td><a href="https://connctd.com/">connctd.com</a></td>
         </tr>
+        <tr>
+            <td>Adopter *</td>
+            <td>Paralus</td>
+            <td align="center"><img height="32px" src="../../../static/adopters/paralus.svg" alt="Paralus"></td>
```
Comment (kevgo) [inline templates/repository/common/ADOPTERS.md:182]: Following the structure of the other entries, this should work once this PR is merged:

```suggestion
            <td align="center"><img height="32px" src="[../../../static](https://raw.githubusercontent.com/ory/meta/master/static/adopters/paralus.svg" alt="Paralus"></td>
```

**Review by kevgo — APPROVED** [review]
🚢 once the comment is addressed

File: templates/repository/common/ADOPTERS.md (hunk 1 above)
Comment (vinckr) [reply templates/repository/common/ADOPTERS.md]: yea my bad, context switching too much and forgot to replace the test URL, 
should be fine now!

**Review by kevgo — APPROVED** [review]

**Comment by vinckr** [comment]
I think needs approval from someone with write access.

**Pull request merged**